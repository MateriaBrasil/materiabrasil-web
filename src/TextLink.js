import React from 'react'
import Link from 'react-router-dom/Link'
import Typography from '@material-ui/core/Typography'

export default ({ to, text }) => (
  <div style={{ textAlign: 'center', marginTop: 16 }}>
    <Link to={to}>
      <Typography>{text}</Typography>
    </Link>
  </div>
)
