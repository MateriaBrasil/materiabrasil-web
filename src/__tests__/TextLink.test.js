import React from 'react'
import renderer from 'react-test-renderer'

import TextLink from '../TextLink'

jest.mock('react-router-dom/Link', () => props => (
  <div {...props}>{props.children}</div>
))

it('renders correctly', () => {
  const tree = renderer.create(<TextLink to="foo" text="bar" />).toJSON()
  expect(tree).toMatchSnapshot()
})
