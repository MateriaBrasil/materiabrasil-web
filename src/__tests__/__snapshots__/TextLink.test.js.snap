// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  style={
    Object {
      "marginTop": 16,
      "textAlign": "center",
    }
  }
>
  <div
    to="foo"
  >
    <p
      className="MuiTypography-root-1 MuiTypography-body1-10"
    >
      bar
    </p>
  </div>
</div>
`;
